index.html

```diff
@@ -67,30 +74,55 @@ <h2>
     <h2>
       Model
     </h2>
-    <dl class="definitions" data-sort="" data-cite="vc-data-model">
+
+    <p class="issue" title="Definitions under discussion">
+The goal of the definitions in this section is to re-use or establish
+terminology that is common across a variety of digital credential formats and
+protocols.
+    </p>
+
+    <dl class="definitions" data-sort="" data-cite="vc-data-model-2.0">
       <dt>
         <dfn>Digital credential</dfn>
       </dt>
       <dd>
-        <p>
-          Is a [=verifiable credential=] about a person.
-        </p>
+        A cryptographically secured digital document containing one or more
+        [=claims=] made by an [=issuer=] about one or more [=subjects=]. Examples include
+        ISO mobile driver's licenses [[ISO18013-5]] and W3C [=verifiable
+        credentials=] [[VC-DATA-MODEL-2.0]].
+      </dd>
+      <dt>
+        <dfn>Credential provider</dfn>
+      </dt>
+      <dd>
+        Software that provides a user interface for querying, selecting, and
+        establishing consent for the [=presentation=] of a [=digital
+        credential=]. Examples include a digital wallet, or a password manager,
+        that manages various
+        <a href="#credential-type-examples">credential types</a>.
       </dd>
       <dt>
-        <dfn>Identity credential provider</dfn>
+        <dfn>Presentation request</dfn>
       </dt>
       <dd>
-        An application or service that provides a user interface for selecting
-        and/or querying a [=digital credential=], such as a digital wallet that
-        manages various identity documents and credentials.
+        A format that a [=verifier=] uses, via an [=exchange protocol=], to
+        request a [=digital credential=] from a [=credential provider=].
       </dd>
       <dt>
-        <dfn data-for="digital credential">Request protocol</dfn>
+        <dfn>Presentation</dfn>
       </dt>
       <dd>
-        A standardized format for requesting a [=digital credential=], designed
-        to ensure the secure, private, and interoperable exchange of identity
-        information. See section [[[#protocol-registry]]].
+        A format that a [=credential repository=] uses, via an
+        [=exchange protocol=], to respond to a [=presentation request=] by
+        a [=verifier=].
+      </dd>
+      <dt>
+        <dfn>Exchange protocol</dfn>
+      </dt>
+      <dd>
+        A protocol used for exchanging a [=digital credential=] between a
+        [=credential provider=] and a [=verifier=]. See section
+        [[[#protocol-registry]]].
```

Comment:
Maybe 
```suggestion
        Standardized data structures for [=presentation request|requesting=] and [=presentation response|receiving=] 
        a [=digital credential=] between a [=credential provider=] and a [=verifier=]. See section
        [[[#protocol-registry]]] for supported protocols.
```

Reply:
Hmm, there is more to a protocol than standardized data structures, no? What about this?

```suggestion
        A protocol used for [=presentation request|requesting=] and [=presentation response|receiving=] 
        a [=digital credential=] between a [=credential provider=] and a [=verifier=]. See section
        [[[#protocol-registry]]] for supported protocols.
```

Reply:
> Hmm, there is more to a protocol than standardized data structures, no?

Yeah, you are definitely not wrong, but protocol is somewhat incomplete... the challenge with the formats is that we need to convert them to something that browsers can ingest, and it's unlikely those formats will define (JS compatible) error handling behavior, Web IDL conversions, etc. 

So, what I was trying to convey is that the request can only be a data structure, the browser will do a bunch of security checks and conversion on that data structure (strip out any garbage, reject anything that looks suspect, etc.) and only then will it pass that to the platform.

Reply:
Right, but then what do we call OID4 or VC API or the protocols that define more than just "standardized data structures"?